fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayController.kt

```diff
@@ -266,8 +266,16 @@ class DefaultTabsTrayController(
     }
 
     override fun handleTrayScrollingToPosition(position: Int, smoothScroll: Boolean) {
```

Comment:
I am wondering if this should function name should be closer to the interactor function name. I suspected that we were moving code into the controller that was called by the interactor function above, but the naming didn't quite much up.

Reply:
way ahead of you. I opened a [ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1835102) to handle exactly this once the XML version is removed.